proto/cosmos/protocolpool/v1/types.proto

```diff
@@ -30,4 +30,29 @@ message Budget {
   // For example, if a period is set to 3600, it represents an action that
   // should occur every hour (3600 seconds).
   google.protobuf.Duration period = 8 [(gogoproto.stdduration) = true];
-}
\ No newline at end of file
+}
+
+// ContinuousFund defines the fields of continuous fund proposal.
+message ContinuousFund {
+  // Title is the title of the funds.
+  string title = 1;
+  // Description of the funds.
+  string description = 2;
+  // Recipient address of the account receiving funds.
+  string recipient = 3 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // Metadata is any arbitrary metadata attached.
+  string metadata = 4;
+  // Percentage is the percentage of funds to be allocated from Community pool share on block by block,
+  // till the `cap` is reached or expired.
+  string percentage = 5 [
+    (cosmos_proto.scalar)  = "cosmos.Dec",
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
+    (gogoproto.nullable)   = false
+  ];
+  // Cap is the capital amount, which when its met funds are no longer distributed.
+  cosmos.base.v1beta1.Coin cap = 6;
+  // Optional, if expiry is set, removes the state object when expired.
+  google.protobuf.Timestamp expiry = 7 [(gogoproto.stdtime) = true];
+  // Distributed amount field is to track the distributed amount.
+  cosmos.base.v1beta1.Coin distributed_amount = 8;
```

Comment:
why is there a need to track this? can we check the account is not greater than the cap?

Reply:
there is a chance that funds in account are already used and cap can never be reached if that's the case

Reply:
that is fine, funds are meant to be continous until canceled. So if the cap is reached its stopped, if the cap is never reached it keeps flowing

Reply:
oh, I thought cap is meant to restrict continuous funds flow. I misunderstood that.
Thanks marko, will make changes accordingly